beep/structure.py

```diff
@@ -1016,6 +1048,20 @@ def determine_whether_step_is_discharging(step_dataframe):
     return cap.diff(axis=0).mean(axis=0).diff().iloc[-1] > 0
 
 
+def determine_whether_step_is_charging(step_dataframe):
```

Comment:
Where was this before?  Did it somehow get deleted?

Reply:
Previously, interpolation was only done on discharging steps, so we didn't need this method (to look for charging steps) until now.